src/pages/sns.tsx

```diff
@@ -0,0 +1,480 @@
+                Announcment soon on DSCVR.
+              </motion.p>
+              <motion.p variants={transitions.item}>
+                <Link
+                  className="link-primary-light"
+                  href="https://dscvr.one/p/internet-computer"
+                >
+                  Join the Internet Computer Portal
+                  <ExternalLinkIcon className="inline-block align-bottom ml-2"></ExternalLinkIcon>
+                </Link>
+              </motion.p>
+            </div>
+            <motion.img
+              src={BlobWhite}
+              className="absolute pointer-events-none max-w-none w-[800px] right-[-250px] top-[-150px] md:w-[1500px]  md:right-[-550px] translate-x-[200px] md:top-[-400px]"
+              alt=""
+            />
+          </AnimateSpawn>
+        </section>
+        <AnimateSpawn
+          className="container-10 py-24 md:py-40 relative"
+          variants={transitions.container}
+        >
+          <div className="">
+            <AnimateSpawn
+              el={motion.img}
+              variants={transitions.item}
+              src={BlobBlue}
+              className="absolute pointer-events-none
+              max-w-none
+              w-[1000px]
+              bottom-[-400px]
+              right-[-600px]
+              sm:bottom-[-600px]
+              sm:w-[1200px]
+              sm:right-[-500px]
+            "
+            />
+          </div>
+          <div className="flex flex-col sm:flex-row gap-8 sm:gap-1/10">
+            <motion.div variants={transitions.item} className="flex-1">
+              <motion.h2
+                className="tw-heading-5 md:tw-heading-3 mb-10 md:mb-16"
+                variants={transitions.item}
+              >
+                What an SNS DAO is <span className="text-razzmatazz">NOT</span>
+              </motion.h2>
+              <div className="md:w-8/10">
+                <h3 className="tw-heading-6 md:tw-heading-5 mb-4">
+                  Not an ICO
+                </h3>
+                <p className="tw-paragraph mb-10">
+                  No funds are handed over to developers. The creation of an SNS
+                  is initiated by a decentralized entity (
+                  <Link
+                    className="text-infinite underline hover:no-underline hover:text-black"
+                    href="https://nns.ic0.app/"
+                  >
+                    the NNS
+                  </Link>
+                  ), and ICP raised by the decentralization sale is allocated to
+                  the treasury of the newly born SNS DAO.
+                </p>
+                <h3 className="tw-heading-6 md:tw-heading-5 mb-4">
+                  Not an IPO
+                </h3>
+                <p className="tw-paragraph">
+                  This is not a liquidity exit or a public listing. It’s the
+                  start of a new journey where you as the initial creator and
```

Comment:
Initial creator of what?

It is a bit unclear to me what we mean here.
Specifically, we explicitly don't want to say the creator of the DAO, right?
I think it can currently be misunderstood in this way.

Reply:
Agreed that it is unclear. I will edit to: initial creator of a service.